Buildings/Experimental/DHC/Loads/Examples/CouplingTimeSeries_Hospital.mo

```diff
@@ -0,0 +1,158 @@
+    annotation (Placement(transformation(extent={{100,110},{120,130}})));
+  Buildings.Controls.OBC.CDL.Continuous.MovingMean QAveCooReq_flow(
+    y(
+      unit="W"),
+    final delta=perAve)
+    "Time average of cooling load"
+    annotation (Placement(transformation(extent={{60,-110},{80,-90}})));
+  Buildings.Controls.OBC.CDL.Continuous.MovingMean QAveCooAct_flow(
+    y(
+      unit="W"),
+    final delta=perAve)
+    "Time average of cooling heat flow rate"
+    annotation (Placement(transformation(extent={{100,-110},{120,-90}})));
+equation
+  connect(supHeaWat.T_in,THeaWatSup.y)
+    annotation (Line(points={{-62,44},{-80,44},{-80,40},{-99,40}},color={0,0,127}));
+  connect(TChiWatSup.y,supChiWat.T_in)
+    annotation (Line(points={{-99,-20},{-80,-20},{-80,-16},{-62,-16}},color={0,0,127}));
+  connect(bui.QHea_flow,EHeaAct.u)
+    annotation (Line(points={{30.6667,14.6667},{40,14.6667},{40,60},{90,60},{90,80},{98,80}},color={0,0,127}));
+  connect(bui.QReqHea_flow,EHeaReq.u)
+    annotation (Line(points={{26.6667,-4.66667},{26.6667,-8},{36,-8},{36,80},{58,80}},color={0,0,127}));
+  connect(bui.QReqCoo_flow,ECooReq.u)
+    annotation (Line(points={{28.6667,-4.66667},{28.6667,-60},{58,-60}},color={0,0,127}));
+  connect(bui.QCoo_flow,ECooAct.u)
+    annotation (Line(points={{30.6667,13.3333},{40,13.3333},{40,-40},{90,-40},{90,-60},{98,-60}},color={0,0,127}));
+  connect(bui.QReqHea_flow,QAveHeaReq_flow.u)
+    annotation (Line(points={{26.6667,-4.66667},{26.6667,-7.90323},{35.9677,-7.90323},{35.9677,120},{58,120}},color={0,0,127}));
+  connect(bui.QHea_flow,QAveHeaAct_flow.u)
+    annotation (Line(points={{30.6667,14.6667},{40,14.6667},{40,60},{90,60},{90,120},{98,120}},color={0,0,127}));
+  connect(bui.QReqCoo_flow,QAveCooReq_flow.u)
+    annotation (Line(points={{28.6667,-4.66667},{28.6316,-4.66667},{28.6316,-60},{28.6316,-100},{58,-100}},color={0,0,127}));
+  connect(bui.QCoo_flow,QAveCooAct_flow.u)
+    annotation (Line(points={{30.6667,13.3333},{40,13.3333},{40,-40},{90,-40},{90,-100},{98,-100}},color={0,0,127}));
+  connect(bui.ports_bHeaWat[1],sinHeaWat.ports[1])
+    annotation (Line(points={{30,4},{60,4},{60,20},{120,20}},color={0,127,255}));
+  connect(bui.ports_aHeaWat[1],supHeaWat.ports[1])
+    annotation (Line(points={{10,4},{0,4},{0,40},{-40,40}},color={0,127,255}));
+  connect(supChiWat.ports[1],bui.ports_aChiWat[1])
+    annotation (Line(points={{-40,-20},{0,-20},{0,0},{10,0}},color={0,127,255}));
+  connect(bui.ports_bChiWat[1],sinChiWat.ports[1])
+    annotation (Line(points={{30,0},{60,0},{60,-20},{120,-20}},color={0,127,255}));
+  annotation (
+    Diagram(
+      coordinateSystem(
+        preserveAspectRatio=false,
+        extent={{-160,-140},{160,140}})),
+    __Dymola_Commands(
+      file="modelica://Buildings/Resources/Scripts/Dymola/Experimental/DHC/Loads/Examples/CouplingTimeSeries_Hospital.mos" "Simulate and plot"),
+    experiment(
+      Tolerance=1e-6,
+      StopTime=86400),
```

Comment:
This stop time (86400 seconds) is not the same as the one used in its `.mos` script (which is 604800 seconds).

Reply:
@JayHuLBL fixed it , thanks!
https://github.com/Helarga/modelica-buildings/tree/issue2201_loadsTimeSeries